### PR TITLE
Add android post module to extract subscriber info

### PR DESCRIPTION
**Pull request by Auxilus**
This module extracts phone-related subscriber info such as imei using call service `service call iphonesubinfo <transaction_code>`

### CODE REVIEW

File: modules/post/android/sub_info.rb [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      'IsimDomain',
+      'IsimImpu',
+      'IsimIst',
+      'IsimPcscf',
+      'IsimChallengeResponse',
+      'IccSimChallengeResponse'
+  ]
+    values ||= []
+    arr ||= []
+    for code in 1..29 do
```
[the remark this answers is not in the document]
Comment (Auxilus) [reply modules/post/android/sub_info.rb]: yes 29 is the number of items in `@transaction_codes`

[the remark this answers is not in the document]
Comment (Auxilus) [reply modules/post/android/sub_info.rb]: the command `service call iphonesubinfo ` starts with 1, so I just used `code - 1` to return respective transaction code at line 100

**Comment by jmartin-r7** [comment]
Jenkins test this please.

**Comment by jmartin-r7** [comment]
Jenkins test this please.

**Comment by jmartin-r7** [comment]
Jenkins test this please.

Sorry for the testing noise here.

**Comment by busterb** [comment]
Minor fixes here: 71646da97f

**Comment by busterb** [comment]
## Release Notes

This adds a module for extracting subscriber info on Android phones when there is a privilege-escalated session to the device.

**Comment by Auxilus** [comment]
@busterb thanks for the merge!

**Pull request merged**